gateway_code/control_nodes/cn_no/__init__.py

```diff
@@ -57,9 +57,9 @@ def setup():
         return 0
 
     @staticmethod
-    @logger_call("Control node: Flash")
+    @logger_call("Control node : Flash")
     def flash(*args, **kwargs):
-        """Setup control node."""
+        """ Flash ControlNode """
```

Comment:
Left over from #16 ;) Let's keep the change but remove the spaces before/after,  add a period and replace ControlNode with the previous version (in short, just replace `Setup` with `Flash`) !

Reply:
ok, fixed in last rebase, I think

Reply:
damn, this makes a merge conflict with master, not sure why

Reply:
OK, I'm dumb, I rebased on an outdated master